pywinauto/unittests/test_win32controls.py

```diff
@@ -591,69 +568,64 @@ def testSetEditTextWithUnicode(self):
         #self.ctrl.Select()
         #self.ctrl.SetEditText(u'\u043d\u0435\u0447\u0442', pos_end=2) # u'necht'
         #self.assertEquals(self.ctrl.TextBlock(), u'\u043d\u0435\u0447\u0442\u043e') # u'nechto'
-
+#
 
 class DialogTestCases(unittest.TestCase):
-

```

Comment:
Empty line is required before class docstring. Do not remove them please. This would produce Landscape.io warnings.